deploy/docker/server.py

```
@@ -471,6 +481,59 @@ async def crawl_stream(
+    
+    if not os.path.exists(script_folder):
+        raise HTTPException(400, f"Directory {script_folder} does not exist.")
+    
+    return JSONResponse([file for file in os.listdir(script_folder) if os.path.isfile(os.path.join(script_folder, file))])
+
+@app.post("/scripts/run")
+@limiter.limit(config["rate_limiting"]["default_limit"])
+async def scripts_run(
+    request: Request,
+    body: ScriptRequest,
+    _td: Dict = Depends(token_dep),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)